friends/src/Form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {FormStyle, InputStyle, Submit} from './Styled Components';
+
+const Form = (props) => {
+    return(
+        <FormStyle id="form">
+          <h1>Add a new friend!</h1>
+          <InputStyle onChange={props.change} id='name' type='text' placeholder='name'></InputStyle>
```

Comment:
Here you could use a required param for the input that makes sure the user is entering data and not sending an object to the post request with all or any empty fields